nebula_decoders/include/nebula_decoders/nebula_decoders_hesai/decoders/hesai_decoder.hpp

```diff
@@ -39,48 +41,92 @@ namespace nebula::drivers
 
 struct HesaiDecodeFilteredInfo
 {
-  uint16_t distance_counter = 0;
-  uint16_t fov_counter = 0;
-  uint16_t timestamp_counter = 0;
-  float distance_start = 0;
-  float distance_end = 0;
-  float raw_azimuth_start = 0;
-  float raw_azimuth_end = 0;
-  std::uint32_t packet_timestamp_start = 0;
-  std::uint32_t packet_timestamp_end = 0;
-  NebulaPointCloud point_azimuth_start;
-  NebulaPointCloud point_azimuth_end;
-  NebulaPointCloud point_timestamp_start;
-  NebulaPointCloud point_timestamp_end;
+  uint16_t distance_filtered_count = 0;
+  uint16_t fov_filtered_count = 0;
+  uint16_t timestamp_filtered_count = 0;
+  uint16_t invalid_point_count = 0;
+  uint16_t multiple_return_point_count = 0;
+  uint16_t mutliple_return_point_count = 0;
+  uint16_t total_kept_point_count = 0;
+  uint16_t invalid_packet_count = 0;
+  float cloud_distance_min_m = 0;
+  float cloud_distance_max_m = 0;
+  float cloud_azimuth_min_rad = 0;
+  float cloud_azimuth_max_rad = 0;
+  uint64_t packet_timestamp_min_ns = 0;
```

Comment:
For easier reaability, please make these `deg` instead of `rad` and convert accordingly in the `get_minmax_info` function below.